### DISCUSSION TITLE
Use python-youtube

### DISCUSSION THREAD

**Discussion by dekoza** [ThioJoe/YT-Spammer-Purge#499] [Ideas]
There's this [python-youtube](https://github.com/sns-sdks/python-youtube/) library which basically wraps all YouTube-related API calls in a simple-to-use form :slightly_smiling_face: No need to reinvent the wheel. I'm experimenting with it and hopefully will manage to rewrite parts of your tool using this.

**Comment by ThioJoe** [comment]
No need to reinvent the wheel indeed, considering everything works fine as implemented ;)
